Externals/soundtouch/SoundTouch.cpp

```diff
@@ -520,7 +523,16 @@ uint SoundTouch::receiveSamples(SAMPLETYPE *output, uint maxSamples)
 /// with 'ptrBegin' function.
 uint SoundTouch::receiveSamples(uint maxSamples)
 {
-	uint ret = FIFOProcessor::receiveSamples(maxSamples);
-	samplesOutput += (long)ret;
-	return ret;
+    uint ret = FIFOProcessor::receiveSamples(maxSamples);
+    samplesOutput += (long)ret;
+    return ret;
+}
+
+

```

Comment:
Unnecessary extra blank line?

Reply:
Not sure what you'd want me to do, this is the library as it is, unchanged. A lot of typos and things like this had been fixed, while some new ones introduced apparently. I don't think it's worth fixing, and I was told that dolphin usually doesn't change externals.

Reply:
They actually added it for consistency, as they always have 2 spaces between methods:
https://gitlab.com/soundtouch/soundtouch/-/blob/master/source/SoundTouch/SoundTouch.cpp